diff --git a/dist/esm/bracket-double/calculate-match-position.js b/dist/esm/bracket-double/calculate-match-position.js
index 908dd46dcccb2417a4142d38331232d888724c46..762b499ca0acab1db7f8672b7588b6dbc10cc9e0 100644
--- a/dist/esm/bracket-double/calculate-match-position.js
+++ b/dist/esm/bracket-double/calculate-match-position.js
@@ -18,16 +18,13 @@ export const calculatePositionOfMatchUpperBracket = (rowIndex, columnIndex, { ca
         rowIndex,
         columnIndex,
     });
-    const skipStep = index => Math.floor((index + 1) * 2) - 3;
-    const xResult = columnIndex === 0 || columnIndex === 1
-        ? columnIndex * columnWidth
-        : skipStep(columnIndex) * columnWidth;
+    const xResult = columnIndex * columnWidth;
     return {
         x: xResult + canvasPadding + offsetX,
         y: yResult + canvasPadding + offsetY,
     };
 };
-export const returnLowerBracketColumnIndex = columnIndex => Math.ceil((columnIndex + 1) / 2) - 1;
+export const returnLowerBracketColumnIndex = columnIndex => columnIndex;
 export const calculatePositionOfMatchLowerBracket = (rowIndex, columnIndex, { canvasPadding, rowHeight, columnWidth, offsetX = 0, offsetY = 0 }) => {
     const result = calculateVerticalPositioning({
         rowHeight,
diff --git a/dist/esm/bracket-double/lower-bracket.js b/dist/esm/bracket-double/lower-bracket.js
index b81a5b7724249361deb27623709a4afab3e21bfa..ee38833725adb93ee498312acd4f4e33f470b310 100644
--- a/dist/esm/bracket-double/lower-bracket.js
+++ b/dist/esm/bracket-double/lower-bracket.js
@@ -12,15 +12,12 @@ const LowerBracket = ({ columns, calculatedStyles, gameHeight, gameWidth, onMatc
             rowHeight,
             offsetY: upperBracketHeight,
         });
-        const isUpperSeedingRound = columnIndex % 2 !== 0;
-        const previousBottomPosition = isUpperSeedingRound
-            ? rowIndex
-            : (rowIndex + 1) * 2 - 1;
+        const previousBottomPosition = (rowIndex + 1) * 2 - 1;
         const { previousTopMatch, previousBottomMatch } = getPreviousMatches(columnIndex, columns, previousBottomPosition);
         return (_jsxs("g", { children: [columnIndex !== 0 && (_jsx(ConnectorsLower, Object.assign({}, {
                     bracketSnippet: {
                         currentMatch: match,
-                        previousTopMatch: !isUpperSeedingRound && previousTopMatch,
+                        previousTopMatch,
                         previousBottomMatch,
                     },
                     rowIndex,
diff --git a/dist/esm/bracket-double/lower-connectors.js b/dist/esm/bracket-double/lower-connectors.js
index 19d609079c9caabd9a066fa0c15cd1e507083a7c..0f1a7af29f0145e89fa7b1abfcd9cde475a51f48 100644
--- a/dist/esm/bracket-double/lower-connectors.js
+++ b/dist/esm/bracket-double/lower-connectors.js
@@ -4,29 +4,27 @@ import { getCalculatedStyles } from '../settings';
 import { calculatePositionOfMatchLowerBracket } from './calculate-match-position';
 const ConnectorsLower = ({ bracketSnippet, rowIndex, columnIndex, style, offsetY = 0, }) => {
     const { columnWidth, rowHeight, canvasPadding } = getCalculatedStyles(style);
-    const isUpperSeedingRound = columnIndex % 2 !== 0;
     const currentMatchPosition = calculatePositionOfMatchLowerBracket(rowIndex, columnIndex, {
         canvasPadding,
         rowHeight,
         columnWidth,
         offsetY,
     });
-    const previousBottomPosition = isUpperSeedingRound
-        ? rowIndex
-        : (rowIndex + 1) * 2 - 1;
-    const previousTopMatchPosition = !isUpperSeedingRound &&
+    const previousBottomPosition = (rowIndex + 1) * 2 - 1;
+    const previousTopMatchPosition = bracketSnippet.previousTopMatch &&
         calculatePositionOfMatchLowerBracket(previousBottomPosition - 1, columnIndex - 1, {
             canvasPadding,
             rowHeight,
             columnWidth,
             offsetY,
         });
-    const previousBottomMatchPosition = calculatePositionOfMatchLowerBracket(previousBottomPosition, columnIndex - 1, {
-        canvasPadding,
-        rowHeight,
-        columnWidth,
-        offsetY,
-    });
+    const previousBottomMatchPosition = bracketSnippet.previousBottomMatch &&
+        calculatePositionOfMatchLowerBracket(previousBottomPosition, columnIndex - 1, {
+            canvasPadding,
+            rowHeight,
+            columnWidth,
+            offsetY,
+        });
     return (_jsx(Connectors, { bracketSnippet: bracketSnippet, previousBottomMatchPosition: previousBottomMatchPosition, previousTopMatchPosition: previousTopMatchPosition, currentMatchPosition: currentMatchPosition, style: style }));
 };
 export default ConnectorsLower;
